Source/Initialization/InjectorMomentum.H

```diff
@@ -106,35 +106,36 @@ namespace {
 
         if (u_th == 0._rt) {
             u = u_m; // Trivial case ; avoids division by 0 in the rest of the code below
-        } else if (u_m < 0.6*u_th) {
-            // Mean velocity is lower than thermal velocity
-            // Use the distribution u*exp(-u**2*(1-u_m/u_th)/(2*u_th**2)) as an approximation
+        } else if (std::abs(u_m) < 0.6*u_th) {
```

Comment:
Since `std::abs(u_m)` is used so often, it might be wort storing it in a constant `abs_u_m`?

Reply:
I can make the change, but is this something that the compiler would do anyway, common expression elimination?

Reply:
That's a good question, one would have to test. But it's not too weird to make it a var here.